src/ui/components/component.js

```diff
@@ -236,6 +236,9 @@ export default class Component {
     });
 
     DOM.addClass(this._container, this._className);
+
+    // We use this class to scope some of our styling.
+    DOM.addClass(this._container, 'yxt-Container');
```

Comment:
How about 'yxt-componentContainer',  'yxt-Component-Container', or something like that? A bit more descriptive about what the container div represents.

Reply:
Yes that sgtm! Updating!